packages/blender.rb

```diff
@@ -19,6 +40,8 @@ def self.install
   end
 
   def self.postinstall
-    puts "\nType 'blender' to get started.\n".lightblue
+    puts
+    puts "Type 'blender' to get started.".lightblue
```

Comment:
Would this be better suited as an `ExitMessage`? (not sure exactly what the purpose of this postinstall is, to be honest, but even still it does seem to be better suited as an `ExitMessage`.

Reply:
Made that change... But Blender doesn't actually work. It's there for debugging purposes.

Reply:
Yeah-- just wondering what the value of telling the user to type the name of the package to use the package is, for when blender does start working.

Reply:
Changed it to ExitMessage in any case...